README.md

```diff
@@ -1,4 +1,3 @@
-iv align="center">
   <h1>🧮 math-magicians</h1>
 </div>
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove the Live Demo section since you don't have any working live link in your readme file 😨 So it would be great if you remove the Live Demo section, for now, to make your readme professional 👍 Once you would have a working live link for your project then you can add this Live Demo section again with the working live link in your readme file 😊